pkg/target/config.go

```diff
@@ -0,0 +1,59 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package target
```

Comment:
shouldn't config.go be under a more common package as it applies also for the other commands